app/views/users/index.html.erb

```diff
@@ -1,11 +1,24 @@
 <section class="users-section">
   <ul class="users-list">
     <% @users.each do |user| %>
-      <li>
-        Name: <%= user.name %>
-        <span class="profile-link">
-          <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>
-        </span>
+      <li class = 'form-index-req' >
+        <% if user.id != current_user.id %>
+          <span class="idx-user-name"><%= user.name %> </span>
+          <span class="profile-link">
+            <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>
+          </span>
+
+          <span class="profile-link">
+
+              <%= form_for @friendship do |f| %>
+                <%= hidden_field_tag "confirmer_id", user.id %>
+                <%= hidden_field_tag "requester_id", current_user.id %>
+                <div>
+                  <%= f.submit 'Send Friend Request', class: 'btn-idx-req' %>
+                </div>
+              <% end %>
+          </span>
+        <% end %>
```

Comment:
- [ ] Instead of having the if/end statement create a helper method that has the if and renders a partial that contains the markup within the if/end statement instead of having the logic in the view